data/json/items/magazine/nitrox.json

```diff
@@ -0,0 +1,36 @@
+[
+  {
+    "id": "nitrox_tank_metal",
+    "type": "MAGAZINE",
+    "name": "aluminum nitrox cylinder",
+    "description": "A standard 30 minute aluminum 2216 PSI cylinder for use with a breathing apparatus.",
+    "weight": 6000,
+    "volume": 7,
+    "price": 5600,
+    "material": "superalloy",
```

Comment:
Its name suggests it's made from aluminum?

Reply:
Changed to aluminum.